src/app/component-library/catalog/asset-detail-dialog/asset-detail-dialog.component.html

```diff
@@ -65,14 +65,17 @@
   <!-- Contract Offers -->
   <contract-offer-mini-list
     *ngIf="data.type === 'data-offer'"
-    [contractOffers]="data.dataOffer!.contractOffers"
+    [data]="{
```

Comment:
Is there a reason why we can't do `[dataOffer]="data.dataOffer!"`? I believe creating objects like this is very inefficient from a change-detection perspective